src/dialogsettings.ui

```diff
@@ -373,6 +373,27 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <layout class="QHBoxLayout" name="horizontalLayout">
+         <item>
+          <widget class="QLabel" name="label">
+           <property name="text">
+            <string>Unread change cmd:</string>
```

Comment:
```suggestion
            <string>Unread change command:</string>
```
We always used command in user visible stings. I am obviously not a native speaker, but `command` sounds nicer than `cmd` to me.